setup.cfg

```diff
@@ -0,0 +1,42 @@
+[metadata]
+name = NEURON
+description = Empirically-based simulator for modeling neurons and networks of neurons
+author = Michael Hines, Yale, Blue Brain Project
+author-email = michael.hines@yale.edu
+license = Copyright (c) 2018, Michael Hines
```

Comment:
The license is the three clause BSD. (see the nrn/Copyright file)

Reply:
As a short version we went for `Copyright (c) Michael Hines (BSD compatible)`